src/main/java/org/sasanlabs/service/vulnerability/fileupload/UnrestrictedFileUpload.java

```diff
@@ -379,4 +383,28 @@ public ResponseEntity<GenericVulnerabilityResponseBean<String>> getVulnerablePay
                 true,
                 false);
     }
+
+    @AttackVector(
+            vulnerabilityExposed = {
+                VulnerabilityType.UNCONTROLLED_RESOURCE_CONSUPTION,
+                VulnerabilityType.DENIAL_OF_SERVICE
+            },
+            description = "UNRESTRICTED_FILE_UPLOAD_UNCONTROLLED_RESOURCE_CONSUPTION",
+            payload = "UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_10")
+    @VulnerableAppRequestMapping(
+            value = LevelConstants.LEVEL_10,
+            htmlTemplate = "LEVEL_1/FileUpload",
+            requestMethod = RequestMethod.POST)
+    public ResponseEntity<GenericVulnerabilityResponseBean<String>> getVulnerablePayloadLevel10(
+            @RequestParam(REQUEST_PARAMETER) MultipartFile file) throws IOException {
+        // stores files in heap memory indefinitely to allow triggering OutOfMemoryError
+        heapMemoryFileStore.add(file.getBytes());
+        return new ResponseEntity<GenericVulnerabilityResponseBean<String>>(
+                new GenericVulnerabilityResponseBean<String>("File accepted.", true),
```

Comment:
we should return the path of file instead of returning `File Accepted` like we are doing this for other levels.